benchmarks/common.py

```diff
@@ -792,6 +792,71 @@ def decay_batch_exp(self, batch_size, factor=0.5, divisor=2):
             out_batch_size = batch_size - 1
         return max(0, int(out_batch_size))
 
+    def measure_latency_and_memory(self, device, model_name, model_iter_fn, backends):
```

Comment:
Minor, but we should make it clear in the API that this is GPU memory, it could easily get conflated/confused/added CPU memory.